### PR TITLE
[res] Add GRU in CircleSchema

### PR DESCRIPTION
**Pull request by BalyshevArtem**
This pr adds CirGru operation in circle_schema.fbs

for issue: #12320
from draft: #12319

ONE-DCO-1.0-Signed-off-by: Artem Balyshev <a.balyshev@samsung.com>

### CODE REVIEW

File: res/CircleSchema/0.7/circle_schema.fbs
Comment (seanshpark) [inline res/CircleSchema/0.7/circle_schema.fbs:1436]: Can you move this above `BCQGatherOptions` ?

**Comment by hseok-oh** [comment]
Do you have any reason to use `Gru`, not `GRU`? 
I recommended `GRU` because it is comes from `Gated Recurrent Unit`'s abbreviation, such as LSTM.

**Comment by glistening** [comment]
In addition, it is the 1st operation which has the prefix `Cir`. Do we need this prefix?

**Comment by BalyshevArtem** [comment]
> In addition, it is the 1st operation which has the prefix `Cir`. Do we need this prefix?

@glistening, @hseok-oh  
In this comment https://github.com/Samsung/ONE/issues/12320#issuecomment-1892822256 we discussed the reason for this decision. The main reason is to avoid conflict with tflite naming in future. I think it is good idea to emphasize that this is a circle operation.

> Do you have any reason to use `Gru`, not `GRU`?

I can change `CirGru` to `CirGRU`, for it is okay both variants, If @seanshpark  doesn't mind :)

**Comment by seanshpark** [comment]
> CirGru to CirGRU

OK with me :)

File: res/CircleSchema/0.7/circle_schema.fbs
Comment (BalyshevArtem) [reply res/CircleSchema/0.7/circle_schema.fbs]: Done

**Comment by BalyshevArtem** [comment]
> > CirGru to CirGRU
> 
> OK with me :)

Done)

**Review by seanshpark — DISMISSED** [review]
LGTM thank you!

**Comment by glistening** [comment]
@BalyshevArtem 

> @glistening, @hseok-oh In this comment [#12320 (comment)](https://github.com/Samsung/ONE/issues/12320#issuecomment-1892822256) we discussed the reason for this decision. 

Basically @hseok-oh and I are working for runtime. I (and perhaps @hseok-oh) usually don't see `CompilerFE` issues and PRs. I see this PR because it is about schema (which is common to compiler and runtime). 

> The main reason is to avoid conflict with tflite naming in future. I think it is good idea to emphasize that this is a circle operation.

I don't think we need to add prefix `Cir` to `GRU`: 

1) `tflite` has not provided `GRU`, and it is not likely to happen in my opinion. (`GRU` is old-fashined op, ...)
2) It is circle schema, we don't need to add `Cir`.

But I don't want to block this PR. It may be merged if it gets @hseok-oh's approval.
(For your information, @hseok-oh is off this week.)

**Comment by seanshpark** [comment]
@BalyshevArtem , as we are adding new Op in current 0.7 version with some circumstances that there is no particular users (other tools or repos), and behind this, frontend has to do some laborious tasks to provide new `mio-circleXX` module and related changes.
 
But recently our internal repo released a package with 0.7 schema.

So, we have to upgrade the version to 0.8 to follow versioning rules.
Can you please add to new schema file in `res` folder?

If your tasks don't have direct relation with `mio-circleXX` module, then you can skip adding `mio-circle08`. I'll handle related tasks after new schema lands.

**Comment by BalyshevArtem** [comment]
> I don't think we need to add prefix `Cir` to `GRU`:

@seanshpark,
What do you think about removing `Cir` prefix? For me both variants are ok)

**Comment by BalyshevArtem** [comment]
> Can you please add to new schema file in `res` folder?

Create 0.8 file in `res` folder and update GRU naming: `CirGRU` -> `GRU`

**Review by seanshpark — APPROVED** [review]
LGTM thank you!

**Review by hseok-oh — APPROVED** [review]
LGTM. Thanks.

**Pull request merged**
